packages/eas-cli/src/commandUtils/pagination.ts

```diff
@@ -27,19 +27,27 @@ const parseFlagInputStringAsInteger = (
   return inputAsNumber;
 };
 
+export const getLimitFlagWithCustomValues = ({
+  defaultTo,
+  limit,
+}: {
+  defaultTo: number;
+  limit: number;
+}): any =>
+  Flags.integer({
+    description: `The number of items to fetch each query. Defaults to ${defaultTo} and is capped at ${limit}. Using a lower value may help increase command speed.`,
```

Comment:
Consider dropping the last sentence, it's a bit long for a help description, plus the default limit should always be small enough that time improvements would be negligible.